docs/pages/build/setup.mdx

```diff
@@ -72,6 +72,12 @@ To configure an Android or an iOS project for EAS Build, run the following comma
 
 To learn more about what happens behind the scenes, see [build configuration process reference](/build-reference/build-configuration).
 
+For development, we recommend creating a [development build](/develop/development-builds/introduction/), which is a debug build of your app and contains the [`expo-dev-client`](/versions/latest/sdk/dev-client/) library. It helps you iterate as quickly as possible and provides a more flexible, reliable, and complete development environment. To install the library, run the following command:
+
+<Terminal cmd={['$ npx expo install expo-dev-client']} />
+
+> **Note:** If you have created your project with `npx react-native`, your project requires further configuration after installing this library. See steps 1 and 2 from [Install `expo-dev-client` in an existing React Native project](/bare/install-dev-builds-in-bare/).
```

Comment:
```suggestion
```

Reply:
we should move any instructions about this to the development build docs if they're not there

Reply:
We do have a collapsible about installing `expo-dev-client` library but it is not descriptive enough about which steps are required. I'll update that collapsible in the development build doc in a follow-up PR.

Link: https://docs.expo.dev/develop/development-builds/create-a-build/#are-you-using-this-library-in-a

Reply:
Follow-up PR created: https://github.com/expo/expo/pull/35815